src/components/Modals/Send/views/Address.tsx

```diff
@@ -65,6 +65,8 @@ export const Address = () => {
       validate: {
         validateAddress: async (rawInput: string) => {
           if (!asset) return
+          // Don't go invalid on initial empty string
+          if (!rawInput.trim()) return true
```

Comment:
Another regression here:

<img width="383" alt="Screenshot 2025-04-01 at 3 25 39 pm" src="https://github.com/user-attachments/assets/4ca7b195-c057-4abd-abe0-147a684b1921" />

To replicate, enter a valid address, remove it, then enter a space.

https://jam.dev/c/1ec7f491-a6f2-4b9d-9901-51f6f28b3687

Reply:
Aaand 8809c60679 as well